ui/src/underlaySelect.tsx

```diff
@@ -13,8 +12,7 @@ export function UnderlaySelect() {
 
   return (
     <>
-      <ActionBar title="Select Dataset" />
-      <Toolbar />
+      <ActionBar title="Select Dataset" backURL={null} />
```

Comment:
Not for this PR, but we should probably come up with another name for Underlay or Dataset so there's no confusion. Maybe concept set too, since I assume that comes from OMOP's concept table and wouldn't really mean anything for another schema.